.github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml

```
@@ -0,0 +1,30 @@
+0name: Feature suggestion
```

"0name" is not a valid key. I'm also wondering if it should be called "Feature Request", but that's a bit of a nit and either is fine.
```suggestion
name: Feature suggestion
```

---

Not sure how that got there 😄